### ISSUE TITLE
shapeless recipes don't work when the quantities are more than required

### ISSUE THREAD

**Issue by BlackCoyote** [DenizenScript/Denizen-For-Bukkit#1189]
which they really should D:

for example an item with:
  `shapeless_recipe: li@i@paper|i@string|i@slime_ball|i@ink_sack`

won't work if you'd put in two or more for any, or all items, which it kinda should though ;(

**Comment by brainscript** [comment]
:heavy_plus_sign::+1:

**Comment by mcmonkey4eva** [comment]
Via `[quantity=2]` or just inserting the item a second time?

**Comment by Fortifier42** [comment]
Replicated like this:
![Problem](http://puu.sh/l44Fh.png)
```
SomeItem:
  type: item
  display name: Random Item
  material: stone
  shapeless_recipe: i@paper|i@carrot_item|i@potato
```

**Comment by mcmonkey4eva** [comment]
OH I was rushing and misread the issue. I see. Will look into.

**Comment by BlackCoyote** [comment]
![a6d568106058d9a7b0dee01c8b7c5475](https://cloud.githubusercontent.com/assets/6047423/13664072/6e48b566-e6a5-11e5-8259-74628292a66a.gif)
issue seems fixed but now this happens

**Comment by mcmonkey4eva** [comment]
Redirecting to #1851

**Comment by mcmonkey4eva** [comment]
Fixed by the resolution to #1851
